webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -56,14 +65,27 @@ class SubscriptionsPage extends Component {
         this.loadData();
       }
     }
+
+    if (hasUpstreamConnection && !prevProps.hasUpstreamConnection) {
```

Comment:
When the page loads it issues a 'ping' (see new method in UpstreamSubscriptionsController) to see if we can reach the upstream. If that API fails, then we won't hit this block. That failure will bubble up as a *single* UI notification.